src/main/c/Objects/pyjmultimethod.c

```diff
@@ -137,12 +132,16 @@ static PyObject* pyjmultimethod_call(PyObject *multimethod,
         PyJMethodObject* method = (PyJMethodObject*) PyList_GetItem(mm->methodList,
                                   methodPosition);
         int parameterCount = PyJMethod_GetParameterCount(method, env);
-        jboolean varargs = java_lang_reflect_Executable_isVarArgs(env, method->rmethod);
-        if ((parameterCount == argsSize) || (varargs && argsSize >= parameterCount - 1)) {
+        if (keywords && PyDict_Size(keywords) > 0 && !method->isKwArgs) {
+            continue;
```

Comment:
Why continue?  Doesn't this mean someone tried to pass kwargs to a method that does not support it?  Shouldn't we set the error indicator and abort?

Reply:
Someone tried to pass kwargs to a multi-method where one potential method in the multimethod doesn't support kwargs. Continue will check the next method in the multimethod, one of them might support kwargs.